### PR TITLE
Implement DISA Firefox STIG V6R3

### PR DESCRIPTION
**Pull request by lenox-joseph**
#### Description:

- Implements the DISA STIG for Mozilla Firefox, V6R3.
- Removes old rule checks that are no longer in force.
- Uses generated Python2/3 scripts to implement remediation. 

#### Rationale:

- The Mozilla Firefox STIG has moved to use a JSON-based administrative policy file instead of mozilla.cfg, etc, for most application changes.
    - This JSON file is more complicated than the existing scripting used for Google Chrome (nested attributes), so the existing mechanisms don't work well.
- These changes bring the checks up to parity and remediation with all checks in V6R3 manual STIG (V6R2 benchmark).
- Changing JSON files via Bash scripts is, in a word, nightmarish, with the most prevalent shell setups only really handling parsing and not edit/writing (and those are a lot of library things to put into a template) via a template.
    - Python 2/3 (script is compatible with both) was chosen as the interpreter is going to be available on just about every distro under the sun (and definitely on the RHEL systems I am targeting).
    - `jq` is not available by default on many distros that I have to support and I don't see a reason to force an additional dependency to the use of these remediations.
- More complete implementation of #8271

### CODE REVIEW

**Comment by lenox-joseph** [comment]
FYI codeclimate here is going to conflict with PEP8 as it wants indentations that are incorrect.

**Comment by lenox-joseph** [comment]
@Mab879 @matejak FYI this is the replacement for #8271

**Review by jan-cerny — CHANGES_REQUESTED** [review]

File: products/firefox/guide/firefox/firefox_policy-verification/rule.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+documentation_complete: true
+
+prodtype: firefox
+
+title: 'Enable Certificate Verification'
+
+description: "Firefox can be configured to prompt the user to choose a certificate\nto present to a website when asked. To enable certificate verification, \nset <tt>security.default_personal_cert</tt> to <tt>Ask Every Time</tt> in an administrative policies file"
```
[the remark this answers is not in the document]
Comment (lenox-joseph) [reply products/firefox/guide/firefox/firefox_policy-verification/rule.yml:7]: @jan-cerny  Thanks for the notice; I'd adapted the existing text and didn't see the control character.

[the remark this answers is not in the document]
Comment (lenox-joseph) [reply products/firefox/guide/firefox/firefox_policy-verification/rule.yml]: @jan-cerny   Fixed, and I went ahead and fixed the other legacy instance in the product.

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**